README.md

```diff
@@ -0,0 +1,32 @@
+# Django development helper bot: ticketbot
+
+This is the bot which replaces the following types of mentions with their URLs:
+
+* `"#nnnnn"` -- Django Trac's ticket #nnnnn
+* `"!nnnn"` or `"PRnnnn"` -- Pull request nnnn submitted to the django/django GitHub repo
+* `"hhhhhhhh"` (7 or more chars) -- Commit with ID hhhhhhh in the django/django GitHub repo
+
+It has a registered user in Freenode: `ticketbot`.
+
+## Configuration
+
+It needs the following env vars:
+
+* `NICKSERV_PASS` -- The bot's Freenode user password
+* `CHANNELS` -- A comma-separated list of channels it will auto-join to
+
+Example, for running it locally:
+
+```
+$ export NICKSERV_PASS=asecret
```

Comment:
asecret -> password?

Add a discussion of creating a virtualenv with the Python version specified in runtime.txt and installing requirements.txt?